js_modules/dagit/packages/core/src/search/__stories__/SearchDialog.stories.tsx

```diff
@@ -1,19 +1,42 @@
-import {Story, Meta} from '@storybook/react/types-6-0';
+import {MockedProvider} from '@apollo/client/testing';
+import {Meta} from '@storybook/react/types-6-0';
 import * as React from 'react';
 
-import {StorybookProvider} from '../../testing/StorybookProvider';
+import {AnalyticsContext, dummyAnalytics} from '../../app/analytics';
 import {SearchDialog} from '../SearchDialog';
+import {
+  buildPrimarySearch,
+  buildPrimarySearchStatic,
+  buildSecondarySearch,
+  buildSecondarySearchStatic,
+} from '../__fixtures__/Search.fixtures';
 
 // eslint-disable-next-line import/no-default-export
 export default {
   title: 'SearchDialog',
   component: SearchDialog,
 } as Meta;
 
-const Template: Story = (props) => (
-  <StorybookProvider>
-    <SearchDialog searchPlaceholder="" {...props} />
-  </StorybookProvider>
+export const BasicSearch = () => (
+  <AnalyticsContext.Provider value={dummyAnalytics()}>
+    <MockedProvider mocks={[buildPrimarySearchStatic(), buildSecondarySearchStatic()]}>
+      <SearchDialog searchPlaceholder="" />
+    </MockedProvider>
+  </AnalyticsContext.Provider>
 );
 
-export const Simple = Template.bind({});
+export const SlowSecondaryQuerySearch = () => (
+  <AnalyticsContext.Provider value={dummyAnalytics()}>
+    <MockedProvider mocks={[buildPrimarySearchStatic(), buildSecondarySearchStatic(10000)]}>
+      <SearchDialog searchPlaceholder="" />
+    </MockedProvider>
+  </AnalyticsContext.Provider>
+);
+
+export const LotsOfAssetsSearch = () => (
+  <AnalyticsContext.Provider value={dummyAnalytics()}>
+    <MockedProvider mocks={[buildPrimarySearch(), buildSecondarySearch(10000)]}>
```

Comment:
Oh this is awesome to have 🙌 

One thing I've noticed recently is that switching tabs in storybook is taking longer and longer - I wonder if it's actually running all this code every time you click something in the left sidebar and move between stories. Will do a bit more testing and see next week...